Web/Components/Pages/Books/BooksListView.razor.css

```
@@ -0,0 +1,16 @@
+::deep .book-list-row {
```

Biome (2.4.4)

[error] Unexpected unknown pseudo-elements: deep

(lint/correctness/noUnknownPseudoElement)